pkg/netconf/netconf_linux.go

```diff
@@ -178,7 +179,12 @@ func ApplyNetworkConfigs(netCfg *NetworkConfig, userSetHostname, userSetDNS bool
 
 	wg := sync.WaitGroup{}
 
-	//apply network config
+	// Don't start any interfaces other than those specified on the command line.
+	// This allows dhcpcd to be started in Master mode.
+	// Then wait for subsequent dhcpcd commands to start each interface as required.
+	runInactiveDhcp()
```

Comment:
Even if eth0 is in static mode and there are no wifi networks, do we still need to run it?

Reply:
I think we need. This has no effect on other processes when eth0 is static mode and there are no wifi networks.
It's only run dhcpcd daemon in Master mode and wait for subsequent dhcpcd command.